### PR TITLE
Remove npm i prior to make build. Fixed few typos, case in the docume…

### PR DESCRIPTION
**Pull request by gpmattoo**
…ntation

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,17 +149,16 @@ PS: If you are contributing to this repo, please make sure to fork the repo, add
 - Install project dependencies.
 
 ```
-make deps
+make deps npm i
```
Comment (elamaran11) [inline README.md:152]: This line is not required.

**Review by elamaran11 — COMMENTED** [review]
@gpmattoo Have a minor comment. Thanks for fixing the doc error.

File: README.md (hunk 1 above)
Comment (gpmattoo) [reply README.md]: Have removed the obsolete command now

File: mkdocs.yml [hunk 2]
```diff
@@ -28,6 +28,8 @@ nav:
     - Single Cluster AWS Native Observability: patterns/single-new-eks-observability-accelerators/single-new-eks-native-observability.md
     - Single Cluster Open Source Observability: patterns/single-new-eks-observability-accelerators/single-new-eks-opensource-observability.md
     - Single Cluster Mixed Observability: patterns/single-new-eks-observability-accelerators/single-new-eks-mixed-observability.md
+    - Single Cluster Graviton Open Source Observability: patterns/single-new-eks-observability-accelerators/single-new-eks-graviton-opensource-observability.md
+    - Single Cluster with no Observability: patterns/single-new-eks-observability-accelerators/single-new-eks-cluster.md
```
Comment (elamaran11) [inline mkdocs.yml:32]: Just say `Single Cluster`. Also please add the one Existing cluster pattern.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@gpmattoo Thankyou for this, one minor comment and we can merge.

File: mkdocs.yml (hunk 2 above)
Comment (gpmattoo) [reply mkdocs.yml]: done!

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**